persistent-sqlite/Database/Persist/Sqlite.hs

```diff
@@ -646,8 +659,8 @@ sqlColumn noRef (Column name isNull typ def gen _cn _maxLen ref) = T.concat
     , mayGenerated gen
     , case ref of
         Nothing -> ""
-        Just ColumnReference {crTableName=table, crFieldCascade=cascadeOpts} ->
-          if noRef then "" else " REFERENCES " <> escapeE table
+        Just ColumnReference {crTableName=table, crSchemaName=schema, crFieldCascade=cascadeOpts} ->
+          if noRef then "" else " REFERENCES " <> escapeES table schema
```

Comment:
We should probably leave a comment mentioning that this is going to be unenforceable if the schema is non-null.